test/e2e/discord_driver_test.go

```diff
@@ -264,6 +273,14 @@ func (d *discordTester) WaitForLastInteractiveMessagePostedEqual(userID, channel
 	})
 }
 
+func (d *discordTester) GetColorByLevel(level config.Level) string {
+	if d.cfg.AttachmentStatus[level] != "" {
+		return d.cfg.AttachmentStatus[level]
+	}
+
+	return ""
```

Comment:
It's enough to use the `DiscordAttachmentColorStatus` in this scope, the method already "separate" us from other drivers
```suggestion
	return DiscordAttachmentColorStatus[level]
```